### DISCUSSION TITLE
What is the right way to keep WFP Cefsharp browser responsive to user?

### DISCUSSION THREAD

**Discussion by lofti198** [cefsharp/CefSharp#4791] [Q&A]
Hi! Using Cefsharp.WPF.Netcore 123.0.60. Everything actually works almost ok, but browser on my form doesnt respond to my user input (neither clicks, nor scrolling). What am I doing wrong? Added the simple playground project to reproduce the behavior https://github.com/lofti198/CefWPF_PG/blob/master/MainWindow.xaml.cs . Thank you in advance!

```
/// <summary>
/// Interaction logic for MainWindow.xaml
/// </summary>
public partial class MainWindow : Window
{
    public MainWindow()
    {
        InitializeComponent();
        browser.LoadUrlAsync("https://www.youtube.com/results?search_query=llm");
    }

    protected override void OnClosed(EventArgs e)
    {
        // Clean up CefSharp resources explicitly
        Cef.Shutdown();
        base.OnClosed(e);
    }
}
```

**Comment by lofti198** [comment]
!!!UPDATE!!! Webpage actually receives my input , but I can see changes just if I do something with window itself (for example - maximize). It seems there is some issue with rerendering

Tryed to set such settings, but it didnt help
```
protected override void OnStartup(StartupEventArgs e)
{
    base.OnStartup(e);

    var settings = new CefSettings();
    // Add any additional settings you might need
    settings.CefCommandLineArgs.Add("disable-gpu", "1");  // Disables GPU hardware acceleration. If experiencing screen flicker or other graphical issues, try removing this line.
    settings.CefCommandLineArgs.Add("touch-events", "enabled");
    settings.CefCommandLineArgs.Add("high-dpi-support", "1");  // Enable High DPI support

    // Initialize Cef with the provided settings
    Cef.Initialize(settings);

}

```

**Reply by amaitland** [reply]
Sounds a lot like https://github.com/cefsharp/CefSharp/issues/3280

If you are using an `Intel Iris Xe` graphics card then you need to upgrade your drivers.

**Reply by lofti198** [reply]
I guess it is not my case. I also checked and found the same behavior on WPF minimal example https://github.com/cefsharp/CefSharp.MinimalExample/tree/master/CefSharp.MinimalExample.Wpf 
![image](https://github.com/cefsharp/CefSharp/assets/42960488/b35c64ba-1c9f-426a-8312-38c59c2e785b)

**Reply by amaitland** [reply]
Did you upgrade your `GPU` driver? If you switch `WPF` to software rendering and it works then it's likely the same `GPU` driver issue.

Alternatively you can switch to https://www.nuget.org/packages/CefSharp.Wpf.HwndHost/ (airspace issues apply)

`MinimalExample` works just fine for me.

![image](https://github.com/cefsharp/CefSharp/assets/307872/f42d5a3b-7c5f-4404-b548-87bbba45e615)
